pebblesdb.patch

```diff
@@ -0,0 +1,33 @@
+diff --git a/CMakeLists.txt b/CMakeLists.txt
+index 8f0f1c3..55af696 100644
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -142,6 +142,9 @@ target_sources(pebblesdb
```

Comment:
I'm wondering what's this file for this pull request? If the patch is for PebblesDB, can you make a pull request to the PebblesDB repo?

Reply:
I provided this patch here since these changes in PebblesDB are needed for compatibility to this project.
PebblesDB as such does not benefit from this patch. The original project also contains a similar patch to LevelDB. 
Do you want me to make a pull request to PebblesDB?